etc/telegraf/telegraf.toml.j2

```diff
@@ -39,7 +39,7 @@
 {% endif %}
 
 {% if db_config %}
-{% if not datadog_api_key %}
+{% if not (datadog_api_key or (appdynamics_enabled and machine_agent_enabled)) %}
```

Comment:
Why do we need this check here ? Isn't the `db_config` check enough?

Reply:
Here I just added additional check for appdynamics to datadog one. If I leave 'db_config' only, the db queries will be performed despite the fact that neither AppDynamics nor datadog are configured. Is it fine?